vue-thacer/src/assets/js/thacer-map-setup-search.js

```diff
@@ -32,11 +32,10 @@ export function setupSearchCeramByText(markerClusterGroupCeram, map) {
             doesCeramObjectPassesInputSearchString(obj.properties, inputSearchString)
           ) {
             let label
-            if (obj.properties.Pi != null) {
+            if (obj.properties.Pi) {
               label = 'Π' + obj.properties.Pi
             } else {
-              label = obj.properties.Inv_Fouille
-
+              label = obj.properties.ID
```

Comment:
You may want to do some one liner in such case
let label = obj.properties.Pi ? 'Π' + obj.properties.Pi : obj.properties.ID
or even with ??
or just like it is now :)

Reply:
Very wise